swagger_server/controllers/operand_controller.py

```diff
@@ -48,4 +68,12 @@ def put_operand(operand_id):  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    if not payload:
+        raise ValidationError(400, 'Couldn\'t parse JSON POST body.')
+    if not payload.get('id'):
+        raise ValidationError(400, 'No id field specified')
+    try:
+        operator = Operand.update(payload).where(id=operand_id).execute()
```

Comment:
I'm not sure this'll work unless we change the payload to be like:
```
payload = {
Operand.expression_id = payload['expression_id'],
Operand.rank = payload['rank'],
<the rest of the fields>
}
```

Instead we might wanna try something more like:
```
operand = Operand.get_by_id(operand_id)
operand.rank = payload['rank']
operand.expression_id = payload['expression_id']
<same with the rest of the fields>
operand.save()
```